src/Razor/src/Microsoft.VisualStudio.RazorExtension/source.extension.vsixmanifest

```diff
@@ -12,14 +12,15 @@
     <AllowClientRole>true</AllowClientRole>
   </Metadata>
   <Installation AllUsers="true" Experimental="true">
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0,)" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0,)">
```

Comment:
Was this change actually required or just slipped in?

Reply:
Talked offline